quickbudget/migrations/0001_initial.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 4.1.7 on 2023-04-06 19:07
```

Comment:
migrations are corrupted. please regenerate them.
the error:
```
It is impossible to change a nullable field 'budget_id' on expense to non-nullable without providing a default. This is because the database needs something to populate existing rows.
```